var/spack/repos/builtin/packages/mpt/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+class mpt(Package):
```

Comment:
Why mpt instead of hpe-mpi?

Reply:
It's debatable, but I had a chat with HPE people and they preferred mpt.

Reply:
Thought is that "HPE MPI" is the official name, but the street name most people know this package by is still "mpt".

Old habits and all.